.github/ISSUE_TEMPLATE/bug.yaml

```
@@ -0,0 +1,94 @@
+name: 🐞 Bug
+description: Report a bug related to the Prometheus operator
+title: "<title>"
+labels: ["kind/bug", "needs-triage"]
+body:
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please search to see if an issue already exists for the bug you encountered.
+    options:
+    - label: I have searched the existing issues
+      required: true
+- type: textarea
+  attributes:
+    label: What happened?
+    description: A concise description of what you're experiencing.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Did you expect to see something different?
```

```suggestion
    label: What did you expect to see?
```

I feel like the question is leading to a "Yes/No" answer, we probably want a more details here